test-data/local/util/latex/report_error.tex

```diff
@@ -1,7 +1,7 @@
 \documentclass[screen,8pt]{beamer}
 \usepackage[T1]{fontenc}
 \usepackage[latin1]{inputenc}
-\usetheme{statoil}
+\usetheme{equinor}
```

Comment:
did you actually check if this theme exists and that the latex can process this file? :smile:

Reply:
No - have not spotted these! where are these themes defined?